scripts/check_types

```diff
@@ -6,6 +6,7 @@ require "open3"
 success = true
 files_with_errors = []
 
+*files = ARGV
 files = Dir.glob("./rbi/**/*").sort if ARGV.empty?
```

Comment:
Maybe better to change this check to be `if files.empty?`

Or better yet, refactor this couple of lines to be something like

```rb
files = ARGV.empty? ? Dir.glob.etc. : ARGV
```

or it's expanded form

```rb
files = if ARGV.empty?
  # blah blah
else
  ARGV
end
```